packages/hdwallet-core/src/ethereum.ts

```diff
@@ -33,7 +33,7 @@ export interface ETHGetAddress {
 
 export interface ETHSignTx {
   /** bip32 path to sign the transaction from */
-  addressNList: BIP32Path;
+  addressNList?: BIP32Path;
```

Comment:
This can't be optional b/c it's what's used to tell the wallet which key to sign with.